Tasks/XamarinTestCloudV1/task.loc.json

```diff
@@ -14,7 +14,7 @@
   "version": {
     "Major": 1,
     "Minor": 234,
```

Comment:
Could you please bump the version of the tasks to 235 since the sprint is almost over?

Reply:
Done